Flow.Launcher/MainWindow.xaml.cs

```diff
@@ -84,8 +84,28 @@ private void OnLoaded(object sender, RoutedEventArgs _)
                             QueryTextBox.SelectAll();
                             _viewModel.LastQuerySelected = true;
                         }
+
                     }
+                    else
+                    {
+                    }
+                }
+                else if (e.PropertyName == nameof(MainViewModel.ProgressBarVisibility))
+                {
+                    Dispatcher.Invoke(() =>
+                    {
+                        if (_viewModel.ProgressBarVisibility == Visibility.Hidden)
+                        {
+                            _progressBarStoryboard.Pause();
+                        }
+                        else
+                        {
+                            _progressBarStoryboard.Resume();
+                        }
+                    }, System.Windows.Threading.DispatcherPriority.Render);
```

Comment:
i think the logic should be this instead:
```
Dispatcher.Invoke(() =>
                    {
                        if (_viewModel.ProgressBarVisibility == Visibility.Visible)
                        {
                            if (Visibility == Visibility.Collapsed || Visibility == Visibility.Hidden)
                                _progressBarStoryboard.Pause();

                            if (Visibility == Visibility.Visible)
                                _progressBarStoryboard.Resume();
                        }
                    }, System.Windows.Threading.DispatcherPriority.Render);
```

When the progressbarvisibilty is visible (which means it is running) and main search window is hidden, then pause, otherwise when main search window is visible, resume. 'Visible is the property for main window's visibility'

Can you check?

Reply:
I agree with you. However, the implementation may need a few change, by making the Visibility changed event to the event called by MainWindow.Visibility. I will do that soon, and you can take a look.